app/scripts/background.js

```diff
@@ -154,6 +158,7 @@ async function initialize () {
   const initLangCode = await getFirstPreferredLangCode()
   await setupController(initState, initLangCode)
   log.debug('MetaMask initialization complete.')
```

Comment:
For the first time can not connect with, the issue that the `initState.NetworkController.provider` is `undefined`, so I think we can have a default value for this issue.

The fix code look likes:
```javascript
let defaultProvider = initState.NetworkController === undefined ? { type: "mainnet" } : initState.NetworkController.provider;
ipfsContent(defaultProvider);
```